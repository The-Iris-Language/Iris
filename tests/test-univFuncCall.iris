.. tests a univ function call using format: "classname.funcname"
class Main of Object () {
  public:
    int main() {
      Animal.bark();
      Dog.bark();
      return 0;
    }
  permit:
  private:
}