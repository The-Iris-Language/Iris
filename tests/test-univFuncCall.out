class Main of Object () {
public:
int main() {
Animal.bark();
Dog.bark();
return 0;
}
permit:
private:
}